### ISSUE TITLE
Developer - Continuous Integration - Develop branch pushes to Staging

### ISSUE THREAD

**Issue by westonmcwhorter** [revelry-foundation/nightingale#20]
## Background
Automatic deployments are more efficient and provide clarity as to what was deployed.

## Scenario: PRs with passing tests get deployed when merged

Given CI tests are set up for the project
And the tests pass for a pull request

- [ ] When the PR is merged
- [ ] Then it is deployed to our server (Staging, Production or otherwise)
- [ ] And the deployment script sends a release notification to Rollbar if it's a production deployment (see below)

### Rollbar Release API

 Toward the beginning of the deploy script, make a ping to notify Rollbar that the deployment has started. It will look something like this:

```bash
curl --request POST \
  --url https://api.rollbar.com/api/1/deploy/ \
  --data "access_token=$ROLLBAR_ACCESS_TOKEN" \
  --data "environment=production" \
  --data "revision=$TRAVIS_COMMIT" \
  --data "comment=$TRAVIS_TAG" \
  --data "status=started"
```

At the end of the deploy script, make another request with `status=succeeded`.

**Comment by westonmcwhorter** [comment]
Standard tooling tasks.

**Comment by iamjoshfrank** [comment]
@vegasgeek - we still blocked on this?

**Comment by vegasgeek** [comment]
@iamjoshfrank Yeah, still blocked. Let's pair up when you have a few minutes to chat and perhaps figure out a solution/workaround.

**Comment by vegasgeek** [comment]
Yes, this is still blocked. Here's the issue:

In a normal setup, we have code living on Github and we use Travis to process the code and then push the code out to WPEngine. For this to happen, WPEngine requires an SSH key from Travis to authenticate. But, I've come to find out that Travis does not generate an SSH key for public repos. So, without a public key stored on WPEngine, when Travis tries to push code, it receives an authentication error and the process ends there.

However, this issue isn't a blocker for our work to get done because I am able to push code from my local box directly to WPEngine. This isn't a great long-term solution, but it's also not a pressing issue at the moment. That said, I'm going to see what I can to do sort out a work around.
